tests/test_run.py

```diff
@@ -23,6 +23,17 @@ def test_run_success():
 
 
 @httpretty.activate
+def test_run_success_with_verify():
+    """Should return response when a success criteria has been reached"""
```

Comment:
This test doesn't really do much - it would be better if it were `"""Should tell requests to skip SSL verification if verify==False"""`, testing whether `session.send()` was called with the `verify` kwarg set to `False`. 

Checking the default behaviour would be great too.

Reply:
I've put something in for this.  Feel free to point out ways to do it better; Python isn't my main language so I'm sure it could be nicer!